apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -676,6 +677,16 @@ export const selectAllFieldsHidden = createSelector(
   allFieldsHidden
 );
 
+export const selectCurrentEmptyModelCard = createSelector(
```

Comment:
super minor, but this naming threw me off a little - maybe `selectIsCurrentModelCardEmpty` (and `isModelCardEmpty` for the util function) since they both return booleans?